Makefile

```diff
@@ -40,6 +42,13 @@ uninstall:
 	rmdir $(DESTDIR)$(SYSCONFDIR)/xdg/uniemoji
 	rm -f $(DESTDIR)$(DATADIR)/ibus/component/uniemoji.xml
 
+deb:
+	make install DESTDIR=./deb-output/
+	cp -r DEBIAN deb-output
+	sed -i "s/<currentversion>/$(CURRENTDATE)/" deb-output/DEBIAN/control
```

Comment:
Maybe the makefile should include a proper version, and if you're building from git, the sha1 can be used, appended to the version.

Reply:
The problem with using sha1 for the version is that dpkg cannot determined which package is newer. How about date of last commit followed by the sha1?

Reply:
why not a proper version number for the deb package? i don't think a new deb should be generated per commit anyway